air/Cargo.toml

```diff
@@ -24,6 +24,8 @@ crypto = { version = "0.6", path = "../crypto", package = "winter-crypto", defau
 fri = { version = "0.6", path = "../fri", package = "winter-fri", default-features = false }
 math = { version = "0.6", path = "../math", package = "winter-math", default-features = false }
 utils = { version = "0.6", path = "../utils/core", package = "winter-utils", default-features = false }
+fixed = "1.20.0"
+typenum = "1.17.0"
```

Comment:
Instead of using these two libraries, I wonder if it would be easier to use [libm](https://crates.io/crates/libm) wrapper. It seems to be even more popular than `fixed` library (7M vs. 180K recent downloads) and seems to provide the [log2](https://docs.rs/libm/latest/libm/fn.log2.html) function we looking for.

I haven't tested it, but maybe `log2` implementation in `libm` is close enough to Rust's standard library that we don't need to do anything else?

Reply:
Actually, using `libm` makes total sense. It simplifies the implementation considerably and reduces the surface of errors. Thank you!